packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -283,22 +283,31 @@ export abstract class AbstractMessageManager<
   }
 
   /**
-   * Sets the message to a new status via a call to this.setMsgStatus and
-   * updates the rawSig field in this.messages.
+   * Sets the message via a call to this.setResult and updates status of the message.
    *
    * @param messageId - The id of the Message to sign.
    * @param rawSig - The data to update rawSig in the message.
    * @param status - The new message status.
    */
   setMessageStatusAndResult(messageId: string, rawSig: string, status: string) {
+    this.setResult(messageId, rawSig);
+    this.setMessageStatus(messageId, status);
+  }
+
+  /**
+   * Sets the message result.
+   *
+   * @param messageId - The id of the Message to sign.
+   * @param rawSig - The data to update rawSig in the message.
+   */
+  setResult(messageId: string, rawSig: string) {
```

Comment:
Can we abstract the argument name too and call it `result` rather than `rawSig`?